src/Engine/GameInput/SelectMetaAction.java

```diff
@@ -2,15 +2,20 @@
 
 import Engine.GameAction;
 import Engine.GameActionSet;
+import Engine.Army;
 import Engine.GameInput.GameInputHandler.InputType;
 
 public class SelectMetaAction extends GameInputState<SelectMetaAction.MetaAction>
 {
   enum MetaAction
   {
-    CO_STATS, CO_INFO, DAMAGE_CHART, SAVE_GAME, CO_ABILITY, QUIT_GAME, END_TURN
+    CO_STATS, CO_INFO, DAMAGE_CHART, SAVE_GAME, CO_ABILITY, QUIT_GAME, SWAP, END_TURN
   }
-  private static final MetaAction[] NO_ABILITY = {MetaAction.CO_STATS, MetaAction.CO_INFO, MetaAction.DAMAGE_CHART, MetaAction.SAVE_GAME, MetaAction.QUIT_GAME, MetaAction.END_TURN};
+  // Four different cases isn't great, but I like the relative simplicity for now
+  // Will probably just make a pruned copy on-the-fly if we end up needing more cases
+  private static final MetaAction[] DEFAULT      = {MetaAction.CO_STATS, MetaAction.CO_INFO, MetaAction.DAMAGE_CHART, MetaAction.SAVE_GAME, MetaAction.QUIT_GAME, MetaAction.END_TURN};
+  private static final MetaAction[] ONLY_ABILITY = {MetaAction.CO_STATS, MetaAction.CO_INFO, MetaAction.DAMAGE_CHART, MetaAction.SAVE_GAME, MetaAction.CO_ABILITY, MetaAction.QUIT_GAME, MetaAction.END_TURN};
+  private static final MetaAction[] ONLY_SWAP    = {MetaAction.CO_STATS, MetaAction.CO_INFO, MetaAction.DAMAGE_CHART, MetaAction.SAVE_GAME, MetaAction.QUIT_GAME, MetaAction.SWAP, MetaAction.END_TURN};
```

Comment:
Why have CO ability before Quit game and Swap after it? I'd probably put Swap right after CO ability.

Reply:
Swap is logically a different kind of end turn, not a kind of CO ability.
I don't recall feeling strongly about it, though.

Reply:
The idea is:
menu->tap up->enter = end turn
menu->tap up twice->enter = swap CO